cert_manager/dcv.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""Define the cert_manager.dcv.DomainControlValidation class."""
+
+from requests.exceptions import HTTPError
+
+from cert_manager._endpoint import Endpoint
+
+
+class DomainControlValidation(Endpoint):
+    """Query the Sectigo Cert Manager REST API for Domain Control Validation (DCV) data."""
+
+    def __init__(self, client, api_version="v1"):
+        """Initialize the class.
+
+        :param object client: An instantiated cert_manager.Client object
+        :param string api_version: The API version to use; the default is "v1"
+        """
+        super().__init__(client=client, endpoint="/dcv", api_version=api_version)
+
+    def search(self, **kwargs):
+        """Search the DCV statuses of domains.
+
+        :param dict kwargs the following search keys are supported:
+            position, size, domain, org, department, dcvStatus, orderStatus, expiresIn
+
+        See
+        https://www.sectigo.com/uploads/audio/Certificate-Manager-20.1-Rest-API.html#resources-dcv-statuses
+
+        :return list: a list of DCV statuses
+        """
+        url = self._url("validation")
+        result = self._client.get(url, params=kwargs)
+
+        return result.json()
+
+    def get_validation_status(self, domain=str):
```

Comment:
I think this is supposed to be a type hint, so it should be `domain: str`

Reply:
Good catch, thanks!